src/Bicep.LangServer/Handlers/BicepCodeActionHandler.cs

```diff
@@ -181,8 +187,13 @@ private static CommandOrCodeAction CreateEditLinterRuleAction(DocumentUri docume
             return new CodeAction
             {
                 Title = String.Format(LangServerResources.EditLinterRuleActionTitle, ruleName),
-                Command = Command.Create(LanguageConstants.EditLinterRuleCommandName, documentUri, ruleName, bicepConfigFilePath ?? string.Empty /* (passing null not allowed) */)
```

Comment:
So I guess it's a bug in OmniSharp and VSCode that title is not being enforced as required.  And apparently VS fails mysteriously if not done, huh?

Reply:
My concern is, how do we make sure people adding new Commands get this correct?  Is there a way to enforce it?  At the very least, we could create a new CreateCommand function that requires this value.  But better if we can enforce via tests or compile-time somethow.

Reply:
As discussed offline, created CreateCommand and added comment:
https://github.com/Azure/bicep/commit/b7f80189a73f04a6d45a8662ad27e1f87e80c3c1